<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kerbal Utility Tool 0.1.0</string>
  </property>
  <widget class="QWidget" name="window">
   <widget class="QPushButton" name="combPushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>145</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combComboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>Select a Planet...</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Select a Planet...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kerbin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Duna</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Eve</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Laythe</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="combLineEdit_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="combLineEdit_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="combGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>161</width>
      <height>211</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Combinations Calculator</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="combMassLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Mass (in Kg)</string>
     </property>
    </widget>
    <widget class="QLabel" name="combVeloLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Velocity (in m/s)</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="combTextBrowser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>155</width>
      <height>80</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="veloTextBrowser">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>219</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mk25SpinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="mk16lSpinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="Mk2rSpinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>90</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="mk16SpinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="mk12rSpinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="veloMk16Label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>65</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mk16</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloMk25Label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>155</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mk25</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloMk2rLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>95</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mk2-R</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloMk12rLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>185</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mk12-R</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloMk16xlLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>125</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mk16-XL</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="veloGroupBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Velocity Calculator</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="veloLineEdit_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="veloMassLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mass (in Kg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="veloVeloLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Velocity (in m/s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>For the Velocity Calculator's planet choice use the one in the Combinations Calculator</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="combTextBrowser_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>155</width>
      <height>80</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="combMassLabel_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>106</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Without Simmetry</string>
    </property>
   </widget>
   <widget class="QLabel" name="combMassLabel_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>With Simmetry</string>
    </property>
   </widget>
   <zorder>veloGroupBox</zorder>
   <zorder>combGroupBox</zorder>
   <zorder>combComboBox</zorder>
   <zorder>combLineEdit_1</zorder>
   <zorder>combLineEdit_2</zorder>
   <zorder>combTextBrowser</zorder>
   <zorder>veloTextBrowser</zorder>
   <zorder>mk25SpinBox</zorder>
   <zorder>mk16lSpinBox</zorder>
   <zorder>Mk2rSpinBox</zorder>
   <zorder>mk16SpinBox</zorder>
   <zorder>mk12rSpinBox</zorder>
   <zorder>veloMk16Label</zorder>
   <zorder>veloMk25Label</zorder>
   <zorder>veloMk2rLabel</zorder>
   <zorder>veloMk12rLabel</zorder>
   <zorder>veloMk16xlLabel</zorder>
   <zorder>veloLineEdit_3</zorder>
   <zorder>veloMassLabel</zorder>
   <zorder>veloVeloLabel</zorder>
   <zorder>combPushButton</zorder>
   <zorder>label</zorder>
   <zorder>combTextBrowser_2</zorder>
   <zorder>combMassLabel_2</zorder>
   <zorder>combMassLabel_3</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
